File: Code/js/gcd.js
num1 = prompt("Enter First Number");
num2 = prompt("Enter Second Number");
gcd = 1;

num = (num1 < num2)? num1 : num2;
for (i = 2; i <= num; i++)
{
  if (num1 % i == 0 && num2 % i == 0)
    gcd = i;
}
console.log(gcd)

File: Code/py/gcd.py
num1 = int(input("Enter First Number: "))
num2 = int(input("Enter Second Number: "))
gcd = 1

num = num1 if num1 < num2 else num2
for i in range(2, num + 1):
    if num1 % i == 0 and num2 % i == 0:
        gcd = i
print(gcd)